app/controllers/base_controller.rb

```diff
@@ -3,9 +3,10 @@ class BaseController < ApplicationController
 
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
 
-  after_action :verify_authorized, except: :index
+  after_action :verify_authorized
 
```

Comment:
ah I had assumed (wrong) that the dashboards controller was the 'index' action as they display a list of records. Shouldn't the dashboards controller be index rather then show?